doc/source/computation.rst

```diff
@@ -196,90 +196,118 @@ parameter:
-    is min for :func:`rolling_min`, max for :func:`rolling_max`, median for
-    :func:`rolling_median`, and mean for all other rolling functions.  See
-    :meth:`DataFrame.resample`'s how argument for more information.
+- ``window``: size of moving window
+- ``min_periods``: threshold of non-null data points to require (otherwise
+  result is NA)
+- ``freq``: optionally specify a :ref:`frequency string <timeseries.alias>`
+  or :ref:`DateOffset <timeseries.offsets>` to pre-conform the data to.
 
-These functions can be applied to ndarrays or Series objects:
+We can then call methods on these ``rolling`` objects. These return like-indexed objects:
 
 .. ipython:: python
 
-   ts = pd.Series(np.random.randn(1000), index=pd.date_range('1/1/2000', periods=1000))
-   ts = ts.cumsum()
+   r.mean()
 
-   ts.plot(style='k--')
+.. ipython:: python
 
-   @savefig rolling_mean_ex.png
-   pd.rolling_mean(ts, 60).plot(style='k')
+   s.plot(style='k--')
 
-They can also be applied to DataFrame objects. This is really just syntactic
-sugar for applying the moving window operator to all of the DataFrame's columns:
+   @savefig rolling_mean_ex.png
+   r.mean().plot(style='k')
 
 .. ipython:: python
    :suppress:
 
    plt.close('all')
 
+They can also be applied to DataFrame objects. This is really just syntactic
+sugar for applying the moving window operator to all of the DataFrame's columns:
+
 .. ipython:: python
 
-   df = pd.DataFrame(np.random.randn(1000, 4), index=ts.index,
-                  columns=['A', 'B', 'C', 'D'])
+   df = pd.DataFrame(np.random.randn(1000, 4), index=s.index,
+                     columns=['A', 'B', 'C', 'D'])
    df = df.cumsum()
 
    @savefig rolling_mean_frame.png
-   pd.rolling_sum(df, 60).plot(subplots=True)
+   df.rolling(window=60).sum().plot(subplots=True)
+
+.. _stats.summary:
+
+Method Summary
+~~~~~~~~~~~~~~
 
-The :func:`rolling_apply` function takes an extra ``func`` argument and performs
+We provide a number of the common statistical functions:
+
+.. currentmodule:: pandas.core.window
+
+.. csv-table::
+    :header: "Method", "Description"
+    :widths: 20, 80
+
+    :meth:`~Rolling.count`, Number of non-null observations
+    :meth:`~Rolling.sum`, Sum of values
+    :meth:`~Rolling.mean`, Mean of values
+    :meth:`~Rolling.median`, Arithmetic median of values
+    :meth:`~Rolling.min`, Minimum
+    :meth:`~Rolling.max`, Maximum
+    :meth:`~Rolling.std`, Unbiased standard deviation
+    :meth:`~Rolling.var`, Unbiased variance
+    :meth:`~Rolling.skew`, Unbiased skewness (3rd moment)
+    :meth:`~Rolling.kurt`, Unbiased kurtosis (4th moment)
+    :meth:`~Rolling.quantile`, Sample quantile (value at %)
+    :meth:`~Rolling.apply`, Generic apply
+    :meth:`~Rolling.cov`, Unbiased covariance (binary)
+    :meth:`~Rolling.corr`, Correlation (binary)
+    :meth:`~Window.mean`, Moving window mean function
+    :meth:`~Window.sum`, Moving window sum function
```

Comment:
re-organized. In fact this whole ``win_type`` bizness was a bit obscured in the original impl. May revisit this to make it more clear (e.g. maybe we should expose ``.window`` for this type of computation, rather than having it as part of ``.rolling``)